hexrdgui/hexrd_config.py

```diff
@@ -2764,3 +2771,8 @@ def clean_panel_buffers(self):
                     'status': 0,
                     'value': [0., 0.],
                 }
+
+    def update_recent_state_files(self, new_file):
```

Comment:
I actually switched this from `add` to `update` since we (potentially) do more than just add by clipping the list to 10. But I'm happy to change it back if that makes more sense!